iceoryx_hoofs/source/posix_wrapper/types.cpp

```diff
@@ -45,8 +45,10 @@ int convertToOflags(const OpenMode openMode) noexcept
     case OpenMode::OPEN_OR_CREATE:
         return O_CREAT;
     case OpenMode::EXCLUSIVE_CREATE:
-        return O_CREAT | O_EXCL;
     case OpenMode::PURGE_AND_CREATE:
+        // wrapped inside function so that the user does not have to use bitwise operations; operands have positive
+        // values and result is within integer range
+        // NOLINTNEXTLINE(hicpp-signed-bitwise)
```

Comment:
Besides, we have no choice but to use them as they are defined by POSIX (not sure whether they have to be signed, but they are for some implementations at least). That is how I argued when I encountered a similar situation.

Tthe existing argument here is also fine, essentially both they we have no real choice but using it is not causing problems.